web/src/OverviewActionBar.tsx

```diff
@@ -481,6 +483,14 @@ function FilterTermFieldError({ error }: { error: string }) {
   )
 }
 
+const FilterTermInfoTooltip = styled(TiltInfoTooltip)`
+  margin: 0 ${SizeUnit(1 / 4)};
+
+  & .fillStd {
+    fill: ${Color.grayLight};
+  }
```

Comment:
i _think_ there's a way to pass props to a styled component's styles, but i wasn't able to figure out how to do it in a way that made typescript happy. @hyu do you have any suggestions, or is this style override an acceptable solution?

Reply:
(Also DM'd this, but adding here for reference)

Here the style override (i.e., for margin and color) is minor enough that it seems like you could get away with the same default padding, and I think it’s actually better if color is consistent. 

In general, if there’s a shared component, I prefer if you can change the appearance in all instances just by looking in the one component itself. Vs, like, having to check for custom styles applied in each usage.

(i.e., you create a new prop like you did with `size`, and then inside the shared component you add some conditional style when a certain prop is coming through. styled-components does have that native method you mentioned, but usually we just use the prop to fiddle with the className. Like, if you wanted to change the color you might toggle a class like “isHighlighted” or something.)